charts/airflow/templates/worker/worker-statefulset.yaml

```diff
@@ -162,7 +162,39 @@ spec:
             {{- else }}
             - "exec airflow celery worker"
             {{- end }}
-          {{- if $volumeMounts }}
+          {{- if .Values.worker.livenessProbe.enabled }}
+          livenessProbe:
+            initialDelaySeconds: {{ .Values.worker.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.worker.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.worker.livenessProbe.timeoutSeconds }}
+            failureThreshold: {{ .Values.worker.livenessProbe.failureThreshold }}
+            exec:
+              command:
```

Comment:
A few questions about the proposed probe command:

1. Will this exact command work on both airflow `1.10.12+` AND `2.X.X`?
2. Can you explain what exactly this is checking for, and under what situations it will return non-0 exit code?
3. Is the celery app always called `airflow.executors.celery_executor.app`?